src/test_lib_json/main.cpp

```diff
@@ -1,9 +1,10 @@
-// Copyright 2007-2010 Baptiste Lepilleur
```

Comment:
Sorry, missed this because I apparently am having trouble getting updates from github (they go to my personal email instead of work email.)

The previous thing I did was replace with The JsonCpp authors. I actually missed a bunch of files by accident. Want I should redo it for "Baptiste Lepilleur and the JsonCpp Authors?" globally in a separate PR?

https://github.com/open-source-parsers/jsoncpp/pull/607